### ISSUE TITLE
BUG - sev-3 - Android - Disability rating (individual ratings boxes) are double-announcing, and incorrectly announcing interactive, on screen reader

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#9236]
## What happened?

On my OnePlus I'm seeing the thing with disability rating again (previously fixed in ticket #4000) where it announces the entire box, and then you can swipe through each row in the box (when it should be JUST the box announced). And in addition, for the 'each row' bit, all of them are announcing that they're interactive (see screenshot). Not happening on iOS/that's behaving normally.
<img width="365" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOVVZQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--44963eddda80be16496fcfd35a016d571f8f908e/image.png" alt="image.png" />

## Specs:

- Device:
- OS:
- User Account:
- Accessibility State:

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: High Low 
- Frequency: High Low 

- 3 - Low
- 2 - High
- 1 - Critical

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))

**Comment by TKDickson** [comment]
iOS and Android are behaving identically -- the screen reader reads all of the information in the row, as a single unit, and then swipes to the next row, for individual ratings.

I also double-checked (for fun! thanks to the recent #9298) that when any given field is null, disability ratings screen doesn't crash.

**Comment by TKDickson** [comment]
Confirmed on Android, after merge conflict resolution, that we still are incrementing the in-app review pop-up count on the disability rating screen, and the screen reader still reads it correctly. Ready to merge.

**Comment by TKDickson** [comment]
Doesn't require further testing in develop; closing
